Button.jsx

```diff
@@ -0,0 +1,8 @@
+// src/components/Button.js
+import React from 'react';
+
+const Button = ({ onClick, text }) => {
+  return <button onClick={onClick}>{text}</button>;
+};
```

Comment:
also lets create a folder for tis components. it can be called core

Reply:
Done